packages/backend-common/src/tokens/ServerTokenManager.test.ts

```diff
@@ -13,73 +13,95 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { getVoidLogger } from '../logging/voidLogger';
 import { ConfigReader } from '@backstage/config';
-import { TokenManager } from './types';
 import { ServerTokenManager } from './ServerTokenManager';
+import { Logger } from 'winston';
+import { JWK } from 'jose';
 
 const emptyConfig = new ConfigReader({});
 const configWithSecret = new ConfigReader({
   backend: { auth: { keys: [{ secret: 'a-secret-key' }] } },
 });
+const env = process.env;
+let logger: Logger;
 
 describe('ServerTokenManager', () => {
-  it('should throw if secret in config does not exist', () => {
-    expect(() => ServerTokenManager.fromConfig(emptyConfig)).toThrowError();
+  beforeEach(() => {
+    process.env = { ...env };
```

Comment:
Could we perhaps move this and the associated afterEach down into the development suite? I think it's only necessary there, and that way it's obvious why this extra setup/teardown exists :)

Reply:
Because I ended up needing it in the `authorize` block (for the new test you suggested adding below), I left it here as-is.